health/notifications/alarm-notify.sh.in

```diff
@@ -2191,6 +2312,7 @@ for state in "${SENT_EMAIL}" \
 	"${SENT_TELEGRAM}" \
 	"${SENT_SLACK}" \
 	"${SENT_ROCKETCHAT}" \
+	"${SEND_HANGOUTS}" \
```

Comment:
This should be `SENT_HANGOUTS` not `SEND_HANGOUTS`.

@cakrit this is exactly what I meant couple months ago when reviewing alarm-notify.sh code. There is too many cases where naming convention makes things very error prone.

Reply:
Done. In the previous PR I noticed, but in this one I missed completely. 
As @cakrit said is to many naming conventions, easy to miss.